src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/Azure.Core.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <ProjectGuid>{84491222-6C36-4FA7-BBAE-1FA804129151}</ProjectGuid>
+    <LangVersion>7.2</LangVersion>
```

Comment:
Do we really need to set the lang version explicitly?

Reply:
Yes -- without it I think we default to version C# 7.0

Reply:
What features are we using that require 7.2? In general we should try and avoid setting an explicit version. I think I'd prefer setting LangVersion=latest like we do in corefx in a common property file.

Reply:
I know of at least the `default` literal that we are using.